environments/notifications/in-app-messaging/package.json

```
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "in-app-messaging",
```

```suggestion
  "name": "in-app-messaging"-environment,
```
for consistency with other environments.

---

Fixed in latest commit